README.md

```diff
@@ -247,10 +247,18 @@ These variables are the defaults of our roles, if you want to override the prope
 | smtp_port                      | Port number of SMTP server                                                                                       |                                             |
 | smtp_user                      | Username for SMTP authentication                                                                                 |                                             |
 | smtp_password                  | Password for SMTP authentication                                                                                 |                                             |
-| enable_oidc                    | Enable OIDC integration, this disables the username/password authentication method                           | false                                       |
+| enable_oidc                    | Enable OIDC integration, this disables the username/password authentication method (self exclusive SSO type)     | false                                       |
 | oidc_client_id                 | OIDC client id, required when enable_oidc is true                                                                |                                             |
 | oidc_client_secret             | OIDC client secret, required when enable_oidc is true                                                            |                                             |
 | oidc_server_base_url           | OIDC identity provider base url, required when enable_oidc is true                                               |                                             |
+| enable_oauth2                  | Enable OAUTH2 integration, this disables the username/password authentication method (self exclusive SSO type)   | false                                       |
+| oauth2_client_id               | OAUTH2 client id, required when enable_oauth2 is true                                                            |                                             |
+| oauth2_client_secret           | OAUTH2 client secret, required when enable_oauth2 is true                                                        |                                             |
+| oauth2_server_base_url         | OAUTH2 identity provider base url, required when enable_oauth2 is true                                           |                                             |
```

Comment:
It would not harm to align the naming between helm chart and ansible vars what do you think ? 
https://github.com/trento-project/helm-charts/pull/105/files#diff-4fe47bf048b209721fcc089ff2be62ad2649ba771dec4767cdc690fcc1b02cb5R23

So why not just  `oauth2_base_url` ?

Reply:
We do the same for `oidc`, i just followed what we have